README.md

```diff
@@ -1 +1,40 @@
-# laravel-zerorpc
\ No newline at end of file
+# laravel-zerorpc
+
+This package provides an easy way of connecting to
+[zerorpc](http://www.zerorpc.io/) from a Laravel application. It uses the
+[PHP zerorpc](https://github.com/0rpc/zerorpc-php) client.
+
+## Installation
+
+1. Add facade and providers to `config/app.php`
+
+    ```php
+    'aliases' => [
+        ...
+        'ZeroRPC' => Juwai\LaravelZeroRPC\Facades\ZeroRPC::class,
+    ],
+    ```
+
+    ```php
+    'providers' => [
+        ...
+        Juwai\LaravelZeroRPC\Providers\ZeroRPCContextProvider::class,
+        Juwai\LaravelZeroRPC\Providers\ZeroRPCClientProvider::class,
+    ],
+    ```
+
+1. Publish config file:
+
+    ```bash
+    $ php artisan vendor:publish
+    ```
+
+1. Add real service configuration to the published config file
+`config/zerorpc.php`.
+
```

Comment:
That's up to whoever is using the package. You don't need to configure per environment.